modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -64,6 +64,7 @@ const char* VisualScriptBuiltinFunc::func_name[VisualScriptBuiltinFunc::FUNC_MAX
 	"str2var",
 	"var2bytes",
 	"bytes2var",
+	"color_named",
```

Comment:
@pkowal1982 Or maybe, we should use the elm-lang approach?
```cpp
something =
{ "1"
, "2"
, "3"
};
```
That way we both have no trailing commas, and we don't have a -1+2 change..